### DISCUSSION TITLE
Issues on spacy-experimental?

### DISCUSSION THREAD

**Discussion by BramVanroy** [explosion/spaCy#9947] [Help: Other Questions] [answered]
Hello

I have been working with the experimental UD models for the last couple of days and I like them a lot! I did encounter a bug with multiprocessing, though - related to the new edit tree lemmatizer, which doesn't like to be pickled. I wanted to open a new issue over at the [repository](https://github.com/explosion/spacy-experimental) but the issue and discussion sections are not enabled. Should I post the issue in spaCy's issue tracker, or nowhere at all and wait until spacy-experimental is a bit more mature?

Best

**Comment by polm** [comment, marked as answer]
Posting it here is fine. We have some automation machinery on the main spaCy repo that makes dealing with issues/discussions here easier than spreading them across many repos.

**Reply by BramVanroy** [reply]
Got it! Will try to make a minimal test case when I find the time.
